javaslang/src/main/java/com/google/gwt/core/shared/GwtIncompatible.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2013 Google Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.gwt.core.shared;
```

Comment:
Let's just put it the `javaslang` main package.

What if it is package private, like below - does GWT still process it?

```java
package javaslang;

@interface GwtIncompatible {
}
```